### PR TITLE
App proxy powershell

### PR DESCRIPTION
**Pull request by msmimart**
## Checklist

- [X] This pull request was tested on __both of__:
  - [X] PowerShell 5.1 (Windows)
  - [X] PowerShell 6.x ([Latest PowerShell](https://github.com/PowerShell/PowerShell/releases))
- [X] Scripts do not contain static passwords or other secret tokens.
- [X] All Azure resource identifiers which must be universally unique are guaranteed to be so.

### Testing information

Platform: Windows 10

Powershell version: `$PSVersionTable.PSVersion`

Az version: `(Get-InstalledModule -Name Az).Version`

### CODE REVIEW

File: application-proxy/get-all-connectors/get-all-connectors.ps1 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Get all Azure AD Application Proxy Connector groups with the included connectors
```
[the remark this answers is not in the document]
Comment (sptramer) [reply application-proxy/get-all-connectors/get-all-connectors.ps1]: This script requires an `admin` level of user privileges on the tenant. Make sure that this is clear in a comment at the header.

File: application-proxy/get-all-appproxy-apps/get-all-appproxy-apps.ps1 [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Get all Azure AD Application Proxy applications (AppId, Name of the app, ObjID)
```
[the remark this answers is not in the document]
Comment (sptramer) [reply application-proxy/get-all-appproxy-apps/get-all-appproxy-apps.ps1]: This script requires an admin level of user privileges on the tenant. Make sure that this is clear in a comment at the header.

File: application-proxy/get-all-appproxy-apps-with-policy/get-all-appproxy-apps-with-policy.ps1 [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+   
+  { 
+   Write-Host ("")        
+ 
+   $ITEM.DisplayName + " (AppId: " + $ITEM.AppId+")"; 
+ 
+   Write-Host ("") 
+   Write-Host ("Policy") 
+     
+   Get-AzureADPolicy -Id $POLICY.id | fl
```
[the remark this answers is not in the document]
Comment (sptramer) [reply application-proxy/get-all-appproxy-apps-with-policy/get-all-appproxy-apps-with-policy.ps1]: This cmdlet produces the following error when testing:

```
Get-AzureADPolicy : Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'Id'. Specified
method is not supported.
At C:\Users\sttramer.REDMOND\src\azure-docs-powershell-samples\application-proxy\get-all-appproxy-apps-with-policy\get-
all-appproxy-apps-with-policy.ps1:21 char:26
+    Get-AzureADPolicy -Id $POLICY.id | fl
+                          ~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-AzureADPolicy], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgument,Microsoft.Open.MSGraphBeta.PowerShell.GetPolicy
```

**Comment by sptramer** [comment]
@msmimart Is this PR ready to be re-reviewed?

**Comment by msmimart** [comment]
@sptramer not yet, the dev is reworking the scripts

**Comment by sdwheeler** [comment]
@msmimart What is the status of this PR?

**Comment by msmimart** [comment]
@sdwheeler we have updated scripts now - I'll be adding them to this PR

**Comment by msmimart** [comment]
New scripts are in development. We'll close this PR and open a new one when testing is complete.

**Comment by msmimart** [comment]
#please-close

**Comment by sdwheeler** [comment]
Closing per request.

**Pull request closed without merging**